youtube_dl/extractor/itunes.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import datetime
+import re
+
+from .common import InfoExtractor
+
+from ..utils import (
+    unescapeHTML,
+    int_or_none,
+    unified_strdate,
+    extract_attributes,
+)
+
+
+class iTunesIE(InfoExtractor):
+    _VALID_URL = r'https?://itunes\.apple\.com/(?P<lang>[a-z]{2})/(?P<genre>[a-z0-9-]+)/?(?P<display_id>[a-z0-9-]+)?/(?:id)?(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'https://itunes.apple.com/us/itunes-u/uc-davis-symphony-orchestra/id403834767',
+        'info_dict': {
+            'id': '403834767',
+            'title': 'UC Davis Symphony Orchestra & University Chorus - Free Podcast by UC Davis on iTunes',
+        },
+        'playlist_count': 31,
+    }
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        playlist_id, display_id = mobj.group('id', 'display_id')
+        if not display_id:
+            display_id = playlist_id
+
+        webpage = self._download_webpage(url, display_id)
+
+        video_infos = re.findall(r'var\s+__desc_popup_d_\d+\s*=\s*({[^><]+});', webpage)
+        html_entries = re.findall(r'<tr\s+[^>]*role="row"[^>]+>', webpage)
```

Comment:
It will get the content between `<tr>` and `</tr>`, but not the content inside the tags, like `<tr preview-duration="485000">`